cmd/pullrequest-init/github_test.go

```diff
@@ -56,13 +60,31 @@ func TestGitHubParseURL_errors(t *testing.T) {
 		"https://github.com/owner/repo/pulls/foo",
 	} {
 		t.Run(url, func(t *testing.T) {
-			if o, r, pr, err := parseGitHubURL(url); err == nil {
-				t.Errorf("Expected error, got (%s, %s, %d)", o, r, pr)
+			if o, r, h, pr, err := parseGitHubURL(url); err == nil {
+				t.Errorf("Expected error, got (%s, %s, %s, %d)", o, r, h, pr)
 			}
 		})
 	}
 }
 
+func TestNewGitHubHandler(t *testing.T) {
```

Comment:
The 2 test cases are pretty much identical. Group these into a table driven test: https://github.com/golang/go/wiki/TableDrivenTests

Reply:
I update it.